### DISCUSSION TITLE
The user account for running CKAN

### DISCUSSION THREAD

**Discussion by sneuvonen** [ckan/ckan#7558] [Q&A]
Dear all,
I was wondering what would be the best practice regarding the user account used for running CKAN. The installation and deployment guides does not mention anything about this. A specific account "ckan_admin" or something would make sense, but what kind of privileges should this user have (which directories should it own etc.)? Does it need to be somehow taken into consideration during deployment, e.g. in the WSGI and Supervisor configurations?
Thanks

**Comment by kowh-ai** [comment]
Anything other than root will be fine. What installation guide did you use? The Installation from package guide (https://docs.ckan.org/en/2.10/maintaining/installing/install-from-package.html) has the processes running as the `www-data` user. The deployment guide (https://docs.ckan.org/en/2.10/maintaining/installing/deployment.html) has this user also in the uwsgi configuration.
